source/guide_ackee.rst

```diff
@@ -34,47 +32,81 @@ Ackee_ is released under the `MIT License`_.
 Prerequisites
 =============
 
-Ackee needs a working :lab:`MongoDB <guide_mongodb>` installation. 
-Follow and install MongoDB as written in the Uberlab guide :lab:`here <guide_mongodb>`.
+Ackee_ requires a MongoDB_ database. Set it up using the :lab:`MongoDB Uberlab guide <guide_mongodb>`. You will need your MongoDB_ admin name and password later.
 
-Then setup your domain:
+Set up your domain:
 
 .. include:: includes/web-domain-list.rst
 
-Also install ``yarn`` globally to later use as package manager for :manual:`Node.js <lang-nodejs>`.
+Installation
+============
+
+Install ``yarn`` globally:
 
 .. code-block:: console
 
   [isabell@stardust ~]$ npm install yarn -g
-  /home/isabell/bin/yarn -> /home/isabell/lib/node_modules/yarn/bin/yarn.js
-  /home/isabell/bin/yarnpkg -> /home/isabell/lib/node_modules/yarn/bin/yarn.js
-  + yarn@1.22.4
-  installed 1 package in 1.002s
   [isabell@stardust ~]$ 
 
-Installation
-============
+Clone the `GitHub repository <https://github.com/electerious/Ackee>`_ and install the dependencies:
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ git clone https://github.com/electerious/Ackee
+  [isabell@stardust ~]$ cd Ackee
+  [isabell@stardust Ackee]$ yarn
+
+Configuration
+=============
+
+Ackee Config
+------------
 
-Clone the `GitHub repository of Ackee`_:
+Create a new, empty ``.env`` file in the root of the Ackee_ directory. Add the following content, replacing the placeholders. The environment variables ``ACKEE_USERNAME`` and ``ACKEE_PASSWORD`` define the credentials to log into the web interface. If you want to use Ackee_ to analyze a domain other than ``isabell.uber.space``, you have to add it under ``ACKEE_ALLOW_ORIGIN`` (read more about the CORS headers `in the Ackee documentation <https://github.com/electerious/Ackee/blob/master/docs/CORS%20headers.md>`_). 
 
 .. code-block:: console
 
-  [isabell@stardust ~]$ git clone https://github.com/electerious/Ackee ~/ackee
-  Cloning into '/home/isabell/ackee'...
-  remote: Enumerating objects: 1528, done.
-  remote: Counting objects: 100% (1528/1528), done.
-  remote: Compressing objects: 100% (884/884), done.
-  remote: Total 7991 (delta 1016), reused 1083 (delta 641), pack-reused 6463
-  Receiving objects: 100% (7991/7991), 1.97 MiB | 3.95 MiB/s, done.
-  Resolving deltas: 100% (5471/5471), done.
-  [isabell@stardust ~]$ 
+  ACKEE_MONGODB=mongodb://<admin>_mongoroot:<password>@localhost:27017/admin
+  ACKEE_USERNAME=<ackee_username>
+  ACKEE_PASSWORD=<ackee_password>
+  ACKEE_ALLOW_ORIGIN="https://my-domain-other-than-isabell-on-stardust.de"
+
+
+Web Backend Config
+------------------
+
+.. note::
+
+    Ackee_ is running on port 3000.
```

Comment:
Maybe add a hint on how to change your port in `.env` as `3000` may be already used.

Reply:
As far as I understand it, the baseline for all projects is always a new Uberspace, so that should not be the case. If somebody hosts multiple projects on one space, they should account for the ports.